storage/store/my_receipts_test.go

```diff
@@ -172,3 +172,31 @@ func TestMyExecutionReceiptsStorage(t *testing.T) {
 		})
 	})
 }
+
+func TestMyExecutionReceiptsStorageMultipleStoreInSameBatch(t *testing.T) {
+	withStore := func(t *testing.T, f func(storage.MyExecutionReceipts, storage.ExecutionResults, storage.ExecutionReceipts, storage.DB)) {
```

Comment:
This is only used once, so could just combine this setup with the test case below.